app/workers/enqueue_subject_queue_worker.rb

```diff
@@ -1,36 +1,32 @@
 require 'subjects/cellect_client'
 require 'subjects/postgresql_selection'
+require 'subjects/seen_remover'
 
 class EnqueueSubjectQueueWorker
   include Sidekiq::Worker
 
-  # SGL-PRIORITY
-  # sidekiq_options queue: :high
   sidekiq_options queue: :really_high
 
   sidekiq_options congestion: Panoptes::SubjectEnqueue.congestion_opts.merge({
-    key: ->(workflow_id, user_id, subject_set_id) {
-      "user_#{ workflow_id }_#{user_id}_#{subject_set_id}_subject_enqueue"
+    key: ->(queue_id) {
+      "queue_#{ queue_id }_enqueue"
     }
   })
 
-  attr_reader :workflow, :user, :subject_set_id, :limit, :selection_strategy
+  attr_reader :queue, :workflow, :user, :subject_set_id, :limit, :selection_strategy
 
-  def perform(workflow_id, user_id=nil, subject_set_id=nil, limit=SubjectQueue::DEFAULT_LENGTH, strategy_override=nil)
-    @workflow = Workflow.find(workflow_id)
-    @user = User.find(user_id) if user_id
-    @subject_set_id = subject_set_id
+  def perform(queue_id, limit=SubjectQueue::DEFAULT_LENGTH, strategy_override=nil)
```

Comment:
This'll throw a bunch of errors in Sidekiq when we deploy (and build the AMI, for that matter) for existing jobs. Do we care?